pkg/providers/aws/cluster_network_provider.go

```diff
@@ -51,21 +51,19 @@ import (
 )
 
 const (
-	DefaultRHMIVpcNameTagValue           = "RHMI Cloud Resource VPC"
-	DefaultRHMISubnetNameTagValue        = "RHMI Cloud Resource Subnet"
-	DefaultRHMISecurityGroupNameTagValue = "RHMI Cloud Resource Security Group"
-	DefaultRHMIVpcRouteTableNameTagValue = "RHMI Cloud Resource Route Table"
-	defaultNumberOfExpectedSubnets       = 2
-	clusterOwnedTagKeyPrefix             = "kubernetes.io/cluster/"
-	clusterOwnedTagValue                 = "owned"
-	//network peering
-	tagVpcPeeringNameValue = "RHMI Cloud Resource Peering Connection"
+	clusterOwnedTagKeyPrefix                = "kubernetes.io/cluster/"
+	clusterOwnedTagValue                    = "owned"
+	defaultCIDRMask                         = "26"
+	defaultNumberOfExpectedSubnets          = 2
+	defaultRouteTableNameTagValue           = "Cloud Resource Route Table"
+	defaultSecurityGroupNameTagValue        = "Cloud Resource Security Group"
+	defaultSubnetNameTagValue               = "Cloud Resource Subnet"
+	defaultVpcNameTagValue                  = "Cloud Resource VPC"
+	defaultVpcPeeringConnectionNameTagValue = "Cloud Resource VPC Peering Connection"
```

Comment:
Think we should replace RHMI with RHOAM in the tag values

Reply:
@austincunningham Is CRO only used by RHOAM and does it not make sense to omit RHOAM from the name?

Reply:
My preference is to not refer to any product or operator that uses CRO as this could be used by anyone.

Reply:
yea but `Cloud Resource` is a bit generic how about `CRO Cloud Resource`

Reply:
CRO = Cloud Resource Operator so feels funny to use both acronym and meaning together. e.g. ATM Machine.